app/js/pages/Escrow/index.scss

```diff
@@ -0,0 +1,7 @@
+@import "../../../css//variable-overrides.scss";
+
+.escrow {
+  .escrow--split {
+    background-color: $secondary;
```

Comment:
There is `.bg-secondary` that you can use

Reply:
updated